src/Sarif.Converters/TSLintConverter.cs

```diff
@@ -41,21 +41,22 @@ internal Result CreateResult(TSLintLogEntry entry)
             Result result = new Result()
             {
                 RuleId = entry.RuleName,
-                Message = new Message { Text = entry.Failure }
+                Message = new Message { Text = entry.Failure },
+                Kind = ResultKind.Fail
             };
 
             switch (entry.RuleSeverity)
             {
                 case "ERROR":
-                    result.Level = ResultLevel.Error;
+                    result.Level = FailureLevel.Error;
                     break;
                 case "WARN":
                 case "WARNING":
-                    result.Level = ResultLevel.Warning;
+                    result.Level = FailureLevel.Warning;
                     break;
                 case "DEFAULT":
                 default:
-                    result.Level = ResultLevel.Note;
+                    result.Level = FailureLevel.Note;
```

Comment:
>Note [](start = 48, length = 4)

But if we set `Level` to `Note`, then we should reset `Kind` and not leave it as `Fail`

Reply:
No sir. 'Note' is actually a fail. :)

---
In reply to: [252922438](https://github.com/Microsoft/sarif-sdk/pull/1249#discussion_r252922438) [](ancestors = 252922438)